examples/sample-app/application-template-stibuild.json

```diff
@@ -252,7 +284,39 @@
           "replicas": 1
         },
         "strategy": {
-          "type": "Recreate"
+          "type": "Recreate",
+          "recreateParams": {
+            "pre": {
+              "failurePolicy": "Abort",
+              "execNewPod": {
+                "containerName": "ruby-helloworld-database",
+                "command": [
+                  "/bin/true"
+                ],
+                "env": [
+                  {
+                    "name": "CUSTOM_VAR1",
+                    "value": "custom_value1"
+                  }
+                ]
+              }
+            },
+            "post": {
+              "failurePolicy": "Ignore",
+              "execNewPod": {
+                "containerName": "ruby-helloworld-database",
+                "command": [
+                  "/bin/false"
```

Comment:
Even if failures are being ignored, why do we want /bin/false as the command here?